CHANGELOG.md

```diff
@@ -1,7 +1,29 @@
-## 15.5.0 (April 7, 2017)
+## 15.5.4 (April 11, 2017)
+
+### React Test Renderer
+* Fix compatibility with Enzyme by exposing `batchedUpdates` on shallow renderer. ([@gaearon](https://github.com/gaearon) in [#9382](https://github.com/facebook/react/commit/69933e25c37cf5453a9ef132177241203ee8d2fd).
+
+
+## 15.5.3 (April 7, 2017)
```

Comment:
Great - for now I'll add the same disclaimer from the 'prop-types' repo. Was also thinking of adding the ['yanked' annotation ](http://keepachangelog.com/en/0.3.0/)but since that would be a new convention will propose it in a follow-up PR.